### PR TITLE
Bugfix - hen to Hens, pullet to Pullets

### PR DESCRIPTION
**Pull request by Andrew-Folga**
Changing ingest values for Giles:

hen -> Hens
pullet -> Pullets

### CODE REVIEW

File: test/integration/narrow/routes/concrete-apron.test.js [hunk 1]
```diff
@@ -2,12 +2,9 @@ const { commonFunctionsMock } = require('../../../session-mock')
 const { crumbToken } = require('./test-helper')
 
 describe('Page: /concrete-apron', () => {
-  let varList = {
-    poultryType: '',
-    henVeranda: ''
```
Comment (alanplatt) [inline test/integration/narrow/routes/concrete-apron.test.js:7]: Why did they have this?

File: test/integration/narrow/routes/easy-grip-perches.test.js [hunk 2]
```diff
@@ -61,10 +60,10 @@ describe('Page: /easy-grip-perches', () => {
     const postResponse = await global.__SERVER__.inject(postOptions)
     expect(postResponse.statusCode).toBe(200)
     expect(postResponse.payload).toContain('Select yes if the perches will have a design feature that help the birds grip')
+    delete valList.easyGripPerches
   })
 
   it('user selects eligible option -> store user response and redirect to /building-biosecurity', async () => {
-    valList.easyGripPerches = null
```
Comment (alanplatt) [inline test/integration/narrow/routes/easy-grip-perches.test.js:67]: what was this?

File: test/integration/narrow/routes/mechanical-ventilation.test.js [hunk 3]
```diff
@@ -40,10 +39,11 @@ describe('Page: /mechanical-ventilation', () => {
     const postResponse = await global.__SERVER__.inject(postOptions)
     expect(postResponse.statusCode).toBe(200)
     expect(postResponse.payload).toContain('Select yes if the building will have a mechanical ventilation system with these features')
+    delete valList.mechanicalVentilation
   })
 
   it('user selects eligible option -> store user response and redirect to /hen-ventilation-specification', async () => {
-    valList.mechanicalVentilation = null
```
Comment (alanplatt) [inline test/integration/narrow/routes/mechanical-ventilation.test.js:46]: what was this?

File: test/integration/narrow/routes/pollution-mitigation.test.js [hunk 4]
```diff
@@ -43,11 +41,11 @@ describe('Page: /pollution-mitigation', () => {
     const postResponse = await global.__SERVER__.inject(postOptions)
     expect(postResponse.statusCode).toBe(200)
     expect(postResponse.payload).toContain('Select if the building will have any of the following')
+    delete valList.pollutionMitigation
   })
 
-  it('user selects eligible option -> store user response and redirect to /renewable-energy - hen', async () => {
-    varList.poultryType = 'hen'
-    valList.pollutionMitigation = null
```
Comment (alanplatt) [inline test/integration/narrow/routes/pollution-mitigation.test.js:50]: what was this?

File: test/integration/narrow/routes/renewable-energy.test.js [hunk 5]
```diff
@@ -95,10 +81,10 @@ describe('Page: /renewable-energy', () => {
     const postResponse = await global.__SERVER__.inject(postOptions)
     expect(postResponse.statusCode).toBe(200)
     expect(postResponse.payload).toContain('Select one type of heat exchanger')
+    delete valList.renewableEnergy
   })
 
   it('user selects eligible option -> store user response and redirect to /bird-data-type', async () => {
-    valList.renewableEnergy = null
```
Comment (alanplatt) [inline test/integration/narrow/routes/renewable-energy.test.js:101]: what was this?

File: app/helpers/handlers.js [hunk 6]
```diff
@@ -66,9 +66,10 @@ const insertYarValue = (field, url, request) => {
         return getReplacementText(request, additionalYarKeyName, 'poultry-type', 'poultry-type-A1', 'aviary', 'multi-tier')
       case 'easy-grip-perches':
         return getReplacementText(request, additionalYarKeyName, 'poultry-type', 'poultry-type-A1', 'an aviary\'s', 'a multi-tier system\'s')
+      case 'renewable-energy':
       case 'veranda-features':
-      return field.includes('{{_poultryType_}}') ? getReplacementText(request, additionalYarKeyName, 'poultry-type', 'poultry-type-A1', 'hen', 'pullet') : 
-        getReplacementText(request, 'poultryType', 'poultry-type', 'poultry-type-A1', '30', '10')
+        return field.includes('{{_poultryType_}}') ? getReplacementText(request, additionalYarKeyName, 'poultry-type', 'poultry-type-A1', 'hen', 'pullet') : 
+          getReplacementText(request, 'poultryType', 'poultry-type', 'poultry-type-A1', '30', '10')
```
Comment (ScenarioFactory) [inline app/helpers/handlers.js:72]: Can we remove this change.. it doesn't appear to be a change?

File: app/helpers/handlers.js [hunk 7]
```diff
@@ -66,9 +66,10 @@ const insertYarValue = (field, url, request) => {
         return getReplacementText(request, additionalYarKeyName, 'poultry-type', 'poultry-type-A1', 'aviary', 'multi-tier')
       case 'easy-grip-perches':
         return getReplacementText(request, additionalYarKeyName, 'poultry-type', 'poultry-type-A1', 'an aviary\'s', 'a multi-tier system\'s')
+      case 'renewable-energy':
```
Comment (ScenarioFactory) [inline app/helpers/handlers.js:69]: Is this change intentional.. is it something that came out of test changes?

**Review by alanplatt — APPROVED** [review]

File: test/integration/narrow/routes/concrete-apron.test.js (hunk 1 above)
Comment (Andrew-Folga) [reply test/integration/narrow/routes/concrete-apron.test.js:7]: Unused variable.

**Review by DavidBarker-EE — APPROVED** [review]
